src/EventListener/SchemaFilterListener.php

```diff
@@ -27,7 +27,11 @@ public function __construct(string $configurationTableName)
     /** @var bool */
     private $enabled = false;
 
-    /** @param AbstractAsset|string $asset */
+    /**
+     * @param AbstractAsset|string $asset
+     *
+     * @phpstan-ignore missingType.generics (a listener should receive all types of assets)
```

Comment:
The proper way would be to use `AbstractAsset<*>` (where phpstan will then properly check that the logic actually supports any kind of asset)